core/src/main/java/com/datastax/oss/driver/internal/core/metrics/DropwizardMetricsFactory.java

```diff
@@ -136,36 +128,4 @@ public NodeMetricUpdater newNodeUpdater(Node node) {
       return dropwizardNodeMetricUpdater;
     }
   }
-
-  protected Set<SessionMetric> parseSessionMetricPaths(List<String> paths) {
```

Comment:
Nit: these overridable methods were a hook to add new metrics by subclassing. In particular this is how the DSE driver did it when it was a separate project.
Now that OSS and DSE are unified this is less of a concern, but maybe still nice to have. We can still extract utility methods and simply call them from here.